src/pages/reference/transforms/transforms-introduction.md

```diff
@@ -0,0 +1,297 @@
+---
```

Comment:
This should be your `transforms/index.md` file. Delete the current index file, rename this one, and add the disclaimer at the top. Do whatever you can to make the handler index file to be more technical and look like this.